runkit.example.js

```diff
@@ -0,0 +1,9 @@
+var f = require('futil-js')
```

Comment:
Very small thing - we generally recommend requiring in `futil` as `F` so our runkit example should probably do that too.